### PR TITLE
Add alternative instructions for using a specific Oshinko release

### PR DESCRIPTION
**Pull request by tmckayus**
This change adds simple instructions to generate and process
a modified resources.yaml from https://radanalyticsio/resources.yaml
that uses s2i and oshinko-webui images from a particular Oshinko
release.

### CODE REVIEW

**Comment by tmckayus** [comment]
@elmiko, @mattf, ptal

**Comment by tmckayus** [comment]
hmm, if we include this I should note that the version needs to be >= v0.2.6 or I should go back and tag previous releases.

File: get-started.md [hunk 1]
```diff
@@ -93,3 +93,21 @@ With Oshinko now installed and running in your OpenShift project, we recommend c
 the tutorial applications that the radanalytics.io community have created.
 These tutorials will you show how to deploy and utilize an insightful
 data-driven application with Oshinko and OpenShift.
+
+## 4. <a name="alternative"></a>Alternative commands to run a specific Oshinko release
+
+You can use resources.yaml to install Oshinko components from a specific release beginning with v0.2.6.
```
Comment (elmiko) [inline get-started.md]: can we add code block quotes to the resources.yaml piece, like this

```
You can use `resources.yaml` to ...
```

File: get-started.md [hunk 2]
```diff
@@ -93,3 +93,21 @@ With Oshinko now installed and running in your OpenShift project, we recommend c
...
+First set OSHINKO_VERSION to the release version, for example:
```
Comment (elmiko) [inline get-started.md]: same here with OSHINKO_VERSION, i think we should code block quote it

**Comment by elmiko** [comment]
the text lgtm, just need to get the Gemfile.lock out of this now. (commit is going to delete the lockfile)

**Comment by elmiko** [comment]
lgtm, thanks!

**Pull request merged**